--- a/pyproject.toml	2026-01-19 22:46:07.855937646 +0100
+++ b/pyproject.toml	2026-01-19 22:54:18.388234916 +0100
@@ -67,17 +67,17 @@
 grpcio = "^1.70.0"
 grpcio-health-checking = "^1.70.0"
 protobuf = ">=5.28.0,<7.0.0"
-cryptography = "^44.0.1"
+cryptography = ">=0"
 pycryptodome = "^3.18.0"
 iterators = "^0.0.2"
 typer = ">=0.12.5,<0.21.0"
 tomli = "^2.0.1"
 tomli-w = "^1.0.0"
 pathspec = "^0.12.1"
-rich = "^13.5.0"
+rich = ">=0"
 pyyaml = "^6.0.2"
 requests = "^2.31.0"
-click = "<8.2.0"
+click = ">=0"
 # Optional dependencies (Simulation Engine)
 ray = [
     { version = "==2.51.1", optional = true, python = ">=3.10,<3.13" },
